build-system/tasks/make-extension/template/shared/extensions/amp-__component_name_hyphenated__/__component_version__/storybook/Basic.amp.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import * as Preact from '#src/preact';
+import * as Preact from '#preact';
```

Comment:
Thanks for catching this! Unfortunately we can't run normal import checking on these templates, so it looks like this one slipped by me during review.